lib/match-conditions.js

```diff
@@ -13,6 +13,12 @@ var platform = process.platform;
 var arch = process.arch;
 var distro = '';
 var release = '';
+var fips = '';
+
+if (process.versions.openssl.indexOf('fips') !== -1) {
+  //checks if the node version is fips and is then included in the checks object
```

Comment:
The idea of the comment was to explain why you set `fips = 'fips'` rather than `fips = true`, I still don't understand.